ckanext/digitalassetfields/fanstatic/digitalassetfields_finish.js

```diff
@@ -39,11 +39,28 @@ ckan.module('digitalassetfields_finish', function ($) {
 	 $('#emoji-finish-img').attr('src', $('#emoji-finish-img').data("src"));
 	 $('#emoji-finish-modal').removeClass('animated infinite bounce');
       });
-
+     $('#finish-register-btn').off("click");
       $('#finish-register-btn').on('click', function(e) {
+        console.log("Submit Clicked")
 	 const data = {state: 'submitted', id: ''}
-
-	 setCookie('digitalasset-register', JSON.stringify(data)); 
+	 setCookie('digitalasset-register', JSON.stringify(data));
+	       var target_group_url = window.location.protocol + "//" + window.location.host + "/";
+         var path_parts = window.location.pathname.split('/');
+         var dataset_name = path_parts.pop() || path_parts.pop();
+	       target_group_url = target_group_url + "dataset/groups/" + dataset_name;
+	       console.log("target group url " + target_group_url);
+        var target = document.getElementById('_digitalassetfields_pre-update');
+        $('#emoji-finish-modal').addClass('animated infinite flash');
+	      $('#finish-register-btn').hide();
+        $('#finish-cancel-btn').hide();
+        $.post(target_group_url, { group_added: "6d9337de-8140-4911-9a3f-8c02cdc166fa" },
```

Comment:
Assuming that this is always going to refer to the id of the local group

Reply:
For our purposes this is likely to be true the corresponding group creation logic in our deployment uses this specific id when it creates the groups. The logic is similar to that used when we create the CSIRO organisation. It should because we control initialization in a clean DB and thus can specify the id prior to any other (user) added organizations or groups being created.